releasenotes/notes/add-hpa-deployment-metrics-ksm-corecheck.yaml

```
@@ -0,0 +1,5 @@
+---
+enhancements:
+  - |
+    Added support for `kube_horizontalpodautoscaler_status_current_target_metric` (KSM v2.6.0)
+    and `kube_deployment_status_replicas_ready`.
```

I think we don't need to specify the KSM metrics, but only list the name of the new datadog metrics

```suggestion
    Added `kubernetes_state.hpa.status_target_metric` and `kubernetes_state.deployment.replicas_ready` metrics part of the `kubernetes_state_core` check.
```